api/blockAPI_test.go

```diff
@@ -117,6 +120,58 @@ func receiptsDecompress() ([]map[string]json.RawMessage, error) {
 	return receiptsObject, nil
 }
 
+func blockReceiptsTransform() (map[rpc.BlockNumber][]map[string]json.RawMessage, map[types.Hash][]map[string]json.RawMessage, error) {
+	numResults := make(map[rpc.BlockNumber][]map[string]json.RawMessage)
+	hashResults := make(map[types.Hash][]map[string]json.RawMessage)
+
+	unmodified, err := receiptsDecompress()
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var previousNum rpc.BlockNumber
+
+	for i, item := range unmodified {
+
+		var blockNumber rpc.BlockNumber
+		if err := blockNumber.UnmarshalJSON(item["blockNumber"]); err != nil {
+			log.Error("Cannot unmarshal blockNumber blockReceiptsTransform", "index", i)
+			return nil, nil, err
+		}
+		var blockHash types.Hash
+		json.Unmarshal(item["blockHash"], &blockHash)
+
+		if blockNumber > previousNum {
+			numResults[blockNumber] = []map[string]json.RawMessage{item}
+			hashResults[blockHash] = []map[string]json.RawMessage{item}
+			previousNum = blockNumber
+		} else if blockNumber == previousNum {
+			numResults[blockNumber] = append(numResults[blockNumber], item)
+			hashResults[blockHash] = append(hashResults[blockHash], item)
+		} else {
+			return nil, nil, errors.New(fmt.Sprintf("Expectations violated in blockReceiptsTransform on index %v", i))
+		}
+	}
+
+	return numResults, hashResults, nil
+}
+
+// func TestReceiptFunc(t *testing.T) {
```

Comment:
We can get rid of this entire commented function. If you want to save it to refer to later you could add it to a file in a directory that you don't include in your commits.